absl/time/internal/cctz/include/cctz/civil_time.h

```diff
@@ -150,7 +150,7 @@ namespace cctz {
 //
 // All civil-time types have accessors for all six of the civil-time fields:
 // year, month, day, hour, minute, and second. Recall that fields inferior to
-// the type's aligment will be set to their minimum valid value.
+// the type's alignment will be set to their minimum valid value.
```

Comment:
This file is actually pulled from the CCTZ, which lives in another GitHub repo (https://github.com/google/cctz/blob/master/include/cctz/civil_time.h).  Would you mind reverting this file and making this change upstream?  Abseil will pick up the change when we update our copy of CCTZ.

Reply:
I got it!